packages/editor/src/components/page-attributes/panel.js

```diff
@@ -38,25 +38,24 @@ export function PageAttributesPanel() {
 		return null;
 	}
 
-	const onTogglePanel = ( ...args ) =>
-		toggleEditorPanelOpened( PANEL_NAME, ...args );
```

Comment:
The [`toggleEditorPanelOpened`](https://github.com/WordPress/gutenberg/blob/a99a5b161f210e654ea9e87ccd11ec2f1359e05f/packages/editor/src/store/actions.js#L647-L652) only accepts `panelName` as an argument, so I inlined it.